src/Microsoft.AspNet.Routing/Template/TemplateParser.cs

```diff
@@ -16,6 +16,7 @@ public static class TemplateParser
         private const char EqualsSign = '=';
         private const char QuestionMark = '?';
         private const char Asterisk = '*';
+        private const string PeriodString = ".";
```

Comment:
I need to compare it to a part in the segment which is a string. Then I need to use ToString at that point, which didnt feel right.